documentation/basics/headless.md

```diff
@@ -39,21 +39,21 @@ Windows:
 
 **Note:** the following examples use linux shell syntax, but the options are the same for all platforms.
 
-### Important Notes:
+### Important Notes: {#notes}
```

Comment:
just to clarify, theses headings already have anchors - this here for example: https://www.ilastik.org/documentation/basics/headless#important-notes